### PR TITLE
doc: elaborate on Scylla admin REST API - V2

### PR DESCRIPTION
**Pull request by annastuchlik**
This is V2 of https://github.com/scylladb/scylladb/pull/11849

This commit adds more information about ScyllaDB's REST API, including an example for Docker and a screenshot of the Swagger UI.

### CODE REVIEW

File: docs/operating-scylla/rest.rst [hunk 1]
```diff
@@ -6,6 +6,41 @@ ScyllaDB exposes a REST API to retrieve administrative information from a node a
 administrative operations. For example, it allows you to check or update configuration, 
 retrieve cluster-level information, and more.
 
+The :doc:`nodetool </operating-scylla/nodetool>` CLI tool interacts with a *scylla-jmx* process using JMX. 
+The process, in turn, uses the REST API to interact with the ScyllaDB process.
+
+You can interact with the REST API using :code:`curl`, ScyllaDB's CLI for REST API, or the Swagger UI.
```
Comment (tzach) [inline docs/operating-scylla/rest.rst:12]:
```suggestion
You can interact with the REST API directly using :code:`curl`, ScyllaDB's CLI for REST API, or the Swagger UI.
```

**Comment by annastuchlik** [comment]
@tzach I've merged your suggestion and squashed the commits. Can this PR be merged now?

**Comment by tzach** [comment]
LGTM

**Comment by annastuchlik** [comment]
@scylladb/scylla-maint Please merge.

**Pull request closed without merging**